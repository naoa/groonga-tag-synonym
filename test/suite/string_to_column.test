register commands/tag_synonym

table_create Memos TABLE_HASH_KEY ShortText
column_create Memos tags COLUMN_SCALAR ShortText
column_create Memos synonym COLUMN_SCALAR ShortText

tag_synonym_add --table Memos --column tags

load --table Memos
[
{"_key": "Groonga", "tags": "Groonga"},
{"_key": "Rroonga", "tags": "Groonga Ruby"}
]

select Memos --output_columns 'tags'
