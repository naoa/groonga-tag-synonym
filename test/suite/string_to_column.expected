register commands/tag_synonym
[[0,0.0,0.0],true]
table_create Memos TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Memos tags COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Memos synonym COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
tag_synonym_add --table Memos --column tags
#|e| [tag-synonym] couldn't open synonym column
load --table Memos
[
{"_key": "Groonga", "tags": "Groonga"},
{"_key": "Rroonga", "tags": "Groonga Ruby"}
]
[[0,0.0,0.0],2]
select Memos --output_columns 'tags'
[[0,0.0,0.0],[[[2],[["tags","ShortText"]],["Groonga"],["Groonga Ruby"]]]]
